doc/nrf/app_dev/device_guides/wifi_coex.rst

```diff
@@ -269,11 +245,14 @@ The *DT property* is the name of the devicetree node property that configures th
 
 In cases where the GPIO is asserted after the radio activity has begun, the ``GRANT`` signal triggers a software interrupt, which in turn disables the radio.
 No guarantee is made on the latency of this interrupt, but the ISR priority is configurable.
+Note that the :ref:`nrfxlib:mpsl` (MPSL) library uses interrupts with priority 0.
+This may delay the GPIOTE interrupt in some rare cases.
+For that reason, it's recommended to deny SR radio activity at least 400 microseconds before activity on the other radio, and to use a GPIOTE interrupt priority as close to 0 as possible.
```

Comment:
```suggestion

.. note::
    The :ref:`nrfxlib:mpsl` (MPSL) library uses interrupts with priority 0.
    This may delay the GPIOTE interrupt in some rare cases.
    For that reason, it's recommended to deny SR radio activity at least 400 microseconds before activity on the other radio, and to use a GPIOTE interrupt priority as close to 0 as possible.
```